README.md

```diff
@@ -19,26 +19,47 @@ make
 * podman should be installed on the node.
 * podman should be configured with pull secrets for the images to be scanned.
 
+### Configuration
+
+The binary has a number of built-in configuration files.
+
+A default built-in configuration ([config.toml](./config.toml)) is used if no
+options are specified, and no `./config.toml` file is available from the
+current working directory when running the tool.
+
+A built-in configuration tailored for a specific OpenShift version can be
+specified using `-V`, `--config-for-version` option, for example `-V 4.11`.
+These configuration files are embedded during build time from the directories
+under [dist/releases/](./dist/releases/).
+
+A specific configuration from a file can be specified using
+`--config path/to/config.toml` option.
+
 ### Scan an OpenShift release payload
 
 ```sh
- sudo ./check-payload scan payload \
+ sudo ./check-payload scan payload -V 4.11 \
    --url quay.io/openshift-release-dev/ocp-release:4.11.0-assembly.art6883.4 \
    --output-file report.txt
 ```
 
+Here
+* `-V` specifies the configuration for a particular OpenShift version;
+* `--url` specifies a payload URL;
+* `--output-file` specifies a file to write the scan report to.
+
 ### Scan a container or operator image
 
 ```sh
-sudo ./check-payload scan operator \
+sudo ./check-payload scan operator -V 4.11 \
```

Comment:
For operator, most likely will need to use operator specific config. Initially they should start with no exemptions.

Reply:
As a nit we can use some other example below instead of ocp-priv

Reply:
right